runhedy.py

```diff
@@ -26,8 +26,11 @@ def main():
                 try:
                     level = int(args[2])
                     if level > 8:
```

Comment:
This 8 is now hardcoded (from way back when we had 7 levels) but hedy has a MAX_LEVEL constant that can be used here.